db/sql/0_msar.sql

```diff
@@ -1353,6 +1383,16 @@ DECLARE
 BEGIN
   col_create_defs := msar.process_col_def_jsonb(tab_id, col_defs, raw_default);
   PERFORM __msar.add_columns(__msar.get_relation_name(tab_id), variadic col_create_defs);
+
+  PERFORM
+  __msar.comment_on_column(
+    tab_id,
+    col_create_def.name_,
+    col_create_def.description
+  )
+  FROM unnest(col_create_defs) AS col_create_def
+  WHERE col_create_def.description IS NOT NULL;
```

Comment:
```suggestion
BEGIN
  col_create_defs := msar.process_col_def_jsonb(tab_id, col_defs, raw_default);
  PERFORM __msar.add_columns(fq_table_name, variadic col_create_defs);

  PERFORM
  __msar.comment_on_column(
    fq_table_name,
    col_create_def.name_,
    col_create_def.description
  )
  FROM unnest(col_create_defs) AS col_create_def
  WHERE col_create_def.description IS NOT NULL;
```
Then you'd also have 
```
  fq_table_name text := __msar.get_relation_name(tab_id);
```
in the `DECLARE` block.

This lets you remove one of the private functions, and also makes this function call more similar to the one just above it.